fastjet/internal/delaunay/voronoi.go

```diff
@@ -0,0 +1,343 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package delaunay
+
+import (
+	"fmt"
+	"math"
+)
+
+// Voronoi holds the border information
+type Voronoi struct {
+	maxX float64
+	minX float64
+	maxY float64
+	minY float64
+}
+
+// FIXME can't do any delaunay operation after calling NewVoronoi
+func NewVoronoi(d *Delaunay) *Voronoi {
+	border := make([]*Triangle, 0, len(d.root.A.adjacentTriangles)+len(d.root.B.adjacentTriangles)+len(d.root.C.adjacentTriangles))
```

Comment:
Are you suggesting to copy the first adjacent triangle `copy(border,d.root.A.adjacentTriangles)`, and the rest is appended or all of them? If so how would that be implemented?